lib/resources/CompanyProfile.ts

```diff
@@ -27,4 +52,22 @@ export class CompanyProfileResource extends Resource<
             url: `${this.basePath}`
         })
     }
+
+    public create(createArgs: CompanyProfileCreateAttributes): Promise<CompanyProfile> {
+        return this.makeRequest({
+            method: 'POST',
+            params: _.pick(createArgs, 'companyId'),
+            data: _.omit(createArgs, 'companyId'),
+            url: `${this.basePath}`
+        })
+    }
+
+    public update(id: string | number, updateArgs: CompanyProfileUpdateAttributes): Promise<CompanyProfile> {
```

Comment:
I would remove the `id` argument as well since it's now unused.  Otherwise looks good!

Reply:
removed